etc/cvmfs/osgstorage-auth.conf

```diff
@@ -48,62 +48,13 @@ CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://xcachevirgo.pic.es:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://singapore.nationalresearchplatform.org:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ligo.hpc.swin.edu.au:8443/"
 
-
-distroversion()
-{
-    # systemd-based systems have /etc/os-release
-    # Prefer /etc/os-release because lsb_release isn't always installed
-    #   on systems that have /etc/os-release
-    if [ -f /etc/os-release ]; then
-        # If this function is moved back under etc/cvmfs so it is
-        #   parsed by cvmfs, the '=' will confuse the parser in
-        #   cvmfs versions <= 2.8.1.  Note that under cvmfsexec
-        #   this function does get parsed.
-        sed -n 's/"//g;s/^VERSION_ID=//p' /etc/os-release
-    else
-        lsb_release -rs
-    fi
-}
 CVMFS_AUTHZ_CVMFS_TOKEN_VARNAME=BEARER_TOKEN_FILE
-CVMFS_X509_AUTHZ_LIBS_VALID=false
-if /usr/libexec/cvmfs/authz/cvmfs_x509_validator 2>/dev/null; then
-    CVMFS_X509_AUTHZ_LIBS_VALID=true
-fi
-if [ ! -f /etc/grid-security/certificates/cilogon-basic.pem ] || \
-      [ ! -f /etc/vomses ] || [ ! -d /etc/grid-security/vomsdir/fermilab ] || \
-        ! $CVMFS_X509_AUTHZ_LIBS_VALID; then
-    # Get at least CA certs and voms info files from the oasis repo
-
-    OSGMISDIR="$CVMFS_MOUNT_DIR/oasis.opensciencegrid.org/mis"
-    OSGBASEDIR="$OSGMISDIR/osg-wn-client"
-    X509_CERT_DIR=$OSGMISDIR/certificates
-    if $CVMFS_X509_AUTHZ_LIBS_VALID; then
-        # the needed libraries are already installed
-        # we'll only use portable files, so pick any osg-wn-client
-        CVMFS_AUTHZ_OSG_BASE_DIR="$OSGBASEDIR/current/el8-x86_64"
-    else
-        # we'll use both libraries and portable files
-        # only Redhat Enterprise Linux or derivatives are supported
-        DISTROVERSION="`distroversion|cut -d. -f1`"
-        DISTROARCH="`arch`"
-        # OSGRELEASE=current
-        # choose the release that includes GSI libraries, until LIGO
-        #   is completely switched over to tokens
-        OSGRELEASE=3.5/current
-        if [ "$DISTROVERSION" = 6 ]; then
-            # choose the last OSG release to support el6
-            OSGRELEASE=3.4/current
-        fi
-        CVMFS_AUTHZ_OSG_WN_CLIENT="$OSGBASEDIR/$OSGRELEASE/el$DISTROVERSION-$DISTROARCH"
-    fi
+if [ ! -f /etc/grid-security/certificates/cilogon-basic.pem ]; then
+    # Get CA certs from the oasis repo
+    X509_CERT_DIR="$CVMFS_MOUNT_DIR/oasis.opensciencegrid.org/mis/certificates"
 else
```

Comment:
Can we drop this workaround? Looks like it was [addressed](https://its.cern.ch/jira/browse/CVM-1083) in >= cvmfs 2.4.

Reply:
I thought about it but I'd rather leave it in.  I'm not 100% sure that nobody is still running that old of a client, although since the EL7 end of life there probably isn't anybody.  I'm also not sure how solid a default of /etc/grid-security/certificates is embedded.  It doesn't hurt to keep it in.

Reply:
Oh, and it's also needed because of the `X509_CERT_DIR=` settings in files that include this file, which is required in order for cvmfs to recognize the setting.  I updated the comment to reflect that.